chia/consensus/block_body_validation.py

```diff
@@ -103,18 +105,34 @@ def include_reward_coins(self, block: FullBlock) -> None:
             timestamp = block.foliage_transaction_block.timestamp
             coin_id = coin.name()
             assert coin_id not in self.additions_since_fork
-            self.additions_since_fork[coin_id] = ForkAdd(coin, block.height, timestamp, None, True)
+            self.additions_since_fork[coin_id] = ForkAdd(coin, block.height, timestamp, None, True, False)
 
     def include_spends(self, conds: Optional[SpendBundleConditions], block: FullBlock, header_hash: bytes32) -> None:
         self.update_fork_peak(block, header_hash)
         if conds is not None:
             assert block.foliage_transaction_block is not None
             timestamp = block.foliage_transaction_block.timestamp
             for spend in conds.spends:
-                self.removals_since_fork[bytes32(spend.coin_id)] = ForkRem(bytes32(spend.puzzle_hash), block.height)
+                spend_coin_id = bytes32(spend.coin_id)
```

Comment:
I don't think you need to give this a name. This change is unrelated to the essence of this PR and, I think, makes it harder to read. I would say the same thing about `spend_ph`. It adds changes to the diff, even though it's not a material change. I don't believe these compute any hashes.

Reply:
`spend_coin_id` object was reconstructed for every created coin. Now we only do that once.